windows-installer/generate_sdl_agent_exe.ps1

```diff
@@ -177,6 +188,7 @@ rm -r -Force $RUBY_DEV_DIR
 ##############################
 
 Add-Type -Assembly System.IO.Compression.FileSystem
+rm -Force $STACKDRIVER_ZIP -ErrorAction Ignore
```

Comment:
Can you please move this `rm` above the `Add-Type` (which is setup for the next line)?
Also, using the `rm` alias with the Windows option syntax (`-ErrorAction`) looks weird — can it be `Remove-Item` instead?

Reply:
1. done
2. the line above uses `rm -Force`, so I was being consistent, following your previous feedback.